integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -163,112 +166,60 @@ def _transmit_delete(self, channel_metadata_item_map):
-                    f'Found previously deleted content record while creating record for course: {content_id}'
-                    f'under customer: {self.enterprise_configuration.enterprise_customer.uuid}. Marking record as '
-                    f'active.',
-                    content_id=content_id
-                )
-                past_deleted_transmission.deleted_at = None
-                past_deleted_transmission.channel_metadata = channel_metadata
-                past_deleted_transmission.content_last_changed = content_last_changed
-                past_deleted_transmission.enterprise_customer_catalog_uuid = catalog_uuid
-                past_deleted_transmission.save()
-            else:
-                # Does there exist a record under a different catalog uuid?
-                past_transmission = ContentMetadataItemTransmission.objects.filter(
-                    enterprise_customer=self.enterprise_configuration.enterprise_customer,
-                    plugin_configuration_id=self.enterprise_configuration.id,
-                    integrated_channel_code=self.enterprise_configuration.channel_code(),
-                    content_id=content_id,
-                    deleted_at__isnull=True,
-                ).first()
-                if past_transmission:
-                    self._log_info(
-                        f'Found past content record under another catalog while creating record for course: '
-                        f'{content_id} under customer: {self.enterprise_configuration.enterprise_customer.uuid}. '
-                        f'Updating records customer catalog uuid to {catalog_uuid}.',
-                        content_id=content_id
-                    )
-                    past_transmission.channel_metadata = channel_metadata
-                    past_transmission.content_last_changed = content_last_changed
-                    past_transmission.enterprise_customer_catalog_uuid = catalog_uuid
-                    past_transmission.save()
-                else:
-                    create_transmissions.append(
-                        ContentMetadataItemTransmission(
-                            enterprise_customer=self.enterprise_configuration.enterprise_customer,
-                            integrated_channel_code=self.enterprise_configuration.channel_code(),
-                            content_id=content_id,
-                            channel_metadata=channel_metadata,
-                            content_last_changed=content_last_changed,
-                            enterprise_customer_catalog_uuid=catalog_uuid,
-                            plugin_configuration_id=self.enterprise_configuration.id
-                        )
-                    )
-        ContentMetadataItemTransmission.objects.bulk_create(create_transmissions, batch_size=50, ignore_conflicts=True)
+            transmission.api_response_status_code = response_status_code
+            transmission.api_response_body = response_body
```

Comment:
i've decided to save all response bodies, not just failed bodies. we can revisit this later if it becomes a problem - perhaps make it a tunable behavior to allow verbose debugging on a per-customer basis for investigating.